src/HttpKernel/ContaoCache.php

```diff
@@ -42,4 +47,15 @@ public function fetch(Request $request, $catch = false): Response
     {
         return parent::fetch($request, $catch);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function createStore()
+    {
+        return new Psr6Store([
+            'cache_directory' => $this->cacheDir,
```

Comment:
I think you should use the getter method for this

Reply:
There's no getter for that, it can only be changed via the constructor.

Reply:
`HttpCache` class uses `$this->cacheDir ?: $this->kernel->getCacheDir().'/http_cache'` because apparently the parameter can be `null`. Should we do the same?

Reply:
Done in https://github.com/contao/manager-bundle/pull/60/commits/bc17aa12744a067c77cc8fdd5b619e039ff44fa1.